includes/installer/initSettings.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/* 
+ * dashboard for admin
+ */
+session_start();
+
+if( !( isset( $_SESSION['Username'] ) && isset($_SESSION['Name']) && $_SESSION['Username'] === 'admin' ) )
+{
+	echo "To access this page you need to login as Admin. Please log in first.";
+	die();
+}
+
+
+// Connect to DB
+//
+$sqlConn = new mysqli( $eorgDBserver , $eorgDBuser , $eorgDBpasswd , $eorgDBname );
```

Comment:
__[code_sniffer]__
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Space found before comma in function call

<!-- Comment by [SideCI](https://sideci.com) -->